Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -76,7 +76,7 @@ public class PlayerActivate : MonoBehaviour
         public const float MobileNPCActivationDistance = 256 * MeshReader.GlobalScale;
 
         // Opening and closing hours by building type
-        static byte[] openHours  = {  7,  8,  9,  8,  0,  9, 10, 10,  9,  6,  9, 11,  9,  9,  0,  0, 10, 0 };
+        static byte[] openHours = { 7, 8, 9, 8, 0, 9, 10, 10, 9, 6, 9, 11, 9, 9, 0, 0, 10, 0 };
```

Comment:
Why did you reformat this? It was that way so that it lined up with closeHours.

Reply:
looks like I triggered VS auto-formatting. I can revert this.

Reply:
fixed